portfolio/src/main/webapp/index.html

```diff
@@ -15,62 +17,26 @@ <h1 class="headline-text">Dagmawi Haile</h1>
-            eros magna ac ligula. Curabitur a leo tristique elit pretium suscipit quis eu felis. Ut eros ex, hendrerit ut
-            arcu sit amet, porta tristique neque. Suspendisse nisi dolor, convallis a pretium posuere, feugiat vel sapien. 
-            Sed at est non ante facilisis feugiat rhoncus et elit. Nullam eget justo eget leo 
-            fermentum lacinia. In consequat, nibh ut porttitor consectetur, quam nisl convallis odio, sit amet consequat 
-            eros magna ac ligula. Curabitur a leo tristique elit pretium suscipit quis eu felis. Ut eros ex, hendrerit ut
-            arcu sit amet, porta tristique neque. Suspendisse nisi dolor, convallis a pretium posuere, feugiat vel sapien. 
-            Sed at est non ante facilisis feugiat rhoncus et elit.
-        </p>
-        <p class="about-text"> 
-            Suspendisse nisi dolor, convallis a pretium posuere, feugiat vel sapien. 
-            Sed at est non ante facilisis feugiat rhoncus et elit. Sed sagittis ipsum ut mauris dignissim scelerisque. 
-            Fusce et malesuada tortor. Vivamus eleifend fringilla justo, ultricies sagittis dolor pulvinar et. 
-            Sed vitae libero vel urna ultrices commodo. Cras sagittis mollis suscipit. Nullam eget justo eget leo 
-            fermentum lacinia. In consequat, nibh ut porttitor consectetur, quam nisl convallis odio, sit amet consequat 
-            eros magna ac ligula. Curabitur a leo tristique elit pretium suscipit quis eu felis.
-            Sed vitae libero vel urna ultrices commodo. Cras sagittis mollis suscipit. Nullam eget justo eget leo 
-            fermentum lacinia. In consequat, nibh ut porttitor consectetur, quam nisl convallis odio, sit amet consequat 
-            eros magna ac ligula. Curabitur a leo tristique elit pretium suscipit quis eu felis.Nullam eget justo eget leo 
-            fermentum lacinia.
-        </p>
-      </div>
-    </div>
+    <div class="profile">
+      <h1>About Me</h1>
+        <hr>
+        <div class="profile-wrapper">
+          <div id="headshot"></div>
+          <div class="about-description">
+            <p>
+              I am a third year computer science and electrical engineering students who is interested in designing and
+              creating products that are practicle and innovative. I began my interests in coding starting with websiste
+              design and development in highschool, and now in college I am primarly working with python. I am interested in
+              projects that not only utilize my current skills, but aslo explore the current buzz of the indsutry such as
+              artifical intelligence and machine learning, as well as the interesction of coding and hardware. In my free 
+              time I enjoy playing basketball (Lakers fan), reading, and watching Netflix.
+            </p>
+            <h2 class="details"><span id="first">School:</span> Massachusetts Institue of Techonology</h2>
```

Comment:
Id's need to be unique for the entire page (https://www.w3.org/TR/WCAG20-TECHS/H93.html)

Your javascript and some older browsers will potentially break otherwise.

Use a classname here for "first" instead.
Also some people prefer to use padding after the span instead of adding a literal space after it. Google has no guidance there so your preference.